packaging/docker/Dockerfile

```diff
@@ -79,18 +79,17 @@ RUN mkdir -p /opt/src /var/log/netdata && \
     ln -sf /dev/stdout /var/log/netdata/debug.log && \
     ln -sf /dev/stderr /var/log/netdata/error.log && \
     ln -sf /dev/stdout /var/log/netdata/collector.log && \
-    ln -sf /dev/stdout /var/log/netdata/health.log && \
-    # Add netdata user
-    addgroup --gid ${NETDATA_GID} --system "${DOCKER_GRP}" && \
-    adduser --system --no-create-home --shell /usr/sbin/nologin --uid ${NETDATA_UID} --home /etc/netdata --group "${DOCKER_USR}"
+    ln -sf /dev/stdout /var/log/netdata/health.log
 
 # Long-term this should leverage BuildKit’s mount option.
 COPY --from=builder /app /
 
-# Apply the permissions as described in
+# Add netdata user and apply the permissions as described in
 # https://docs.netdata.cloud/docs/netdata-security/#netdata-directories, but own everything by root group due to https://github.com/netdata/netdata/pull/6543
 # hadolint ignore=DL3013
-RUN chown -R root:root \
+RUN addgroup --gid ${NETDATA_GID} --system "${DOCKER_GRP}" && \
+    adduser --system --no-create-home --shell /usr/sbin/nologin --uid ${NETDATA_UID} --home /etc/netdata --group "${DOCKER_USR}" && \
```

Comment:
This fixes

> adduser: Warning: The home dir /etc/netdata you specified can't be accessed: No such file or directory

(because `/etc/netdata`  is `/app/etc/` before `COPY --from=builder /app /`.

A cosmetic problem, but as any warning it makes you draw attention to it, so decided to fix it.